.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Definitively text files 
+*.txt text
```

Comment:
i guess this will not work. in this case line endings will be changed to native line endings on checkout so if i will have windows os git will change it to CLRF, or no? Maybe we should use smth like *.sh text eol=lf ?

Reply:
yes. u r right. done